### PR TITLE
More alt titles

### PR DESCRIPTION
**Pull request by cebutris**
Adds more alt titles! Looking for more, let me know if you have any ideas!

### CODE REVIEW

File: vorestation.dme [hunk 1]
```diff
@@ -3778,6 +3778,8 @@
 #include "content_arfs\overrides\client\login.dm"
 #include "content_arfs\overrides\client\preference_setup\global\setting_datums.dm"
 #include "content_arfs\overrides\jobs\assistant.dm"
+#include "content_arfs\overrides\jobs\bridge_officer.dm"
```
Comment (cebutris) [inline vorestation.dme]:
```suggestion
```

**Comment by TalkingCactus** [comment]
Let me know when you've stopped adding titles and i'll merge

**Comment by cebutris** [comment]
OH FUCK

**Comment by TalkingCactus** [comment]
> OH FUCK

what

**Comment by TalkingCactus** [comment]
oh yeah you might want to just redo your .dme file

**Comment by cebutris** [comment]
image

**Comment by cebutris** [comment]
cactus can you help me solve my merge conflits without fucking my dme

**Pull request closed without merging**